foreach1.js

```diff
@@ -9,3 +9,11 @@
 var foods = ["pizza", "tacos", "ice cream", "sushi"];
 
 // your code here
+
+for(let i = 0; i < foods.length; i++) {
+    console.log("I like " + foods[i])
+  }
+  
+  foods.forEach(function(food) {
+  //foodLove.push(food);
+  })
```

Comment:
nice work here! watch your indentation, though; lines 17 and 19 should be unindented 1 level